packages/x-tree-view/src/useTreeItem2/useTreeItem2.ts

```diff
@@ -233,6 +282,23 @@ export const useTreeItem2 = <
     };
   };
 
+  const getLabelInputProps = <ExternalProps extends Record<string, any> = {}>(
+    externalProps: ExternalProps = {} as ExternalProps,
+  ): UseTreeItem2LabelInputSlotProps<ExternalProps> => {
+    const externalEventHandlers = extractEventHandlers(externalProps);
+
+    return {
+      ...externalEventHandlers,
+      label,
+      ref: inputRef,
+      'data-element': 'labelInput',
```

Comment:
Is `data-element` used anywhere?

Reply:
Leftover from a blur/focus issue fix exploration 🙈 🤦‍♀️